src/pages/Lend/Filter/LendFilterPage.vue

```diff
@@ -155,6 +158,8 @@ export default {
 			userId: '',
 			filterMenuOpen: false,
 			selectedCustomCategories: {},
+			// eslint-disable-next-line
+			topMessage: `Welcome to Kiva's new filter page! Take it for a spin below, or <a v-kv-track-event="['Lending', 'click-return-classic', 'Exit-CASH-545-2019']" href="https://www.kiva.org/lend?kexpn=lend_filter.lend_filter_versions&kexpv=c">return to the classic view</a> at any time.`,
```

Comment:
@mcstover will `v-kv-track-event` work here with v-html?

Reply:
I think you need to attach an on click handler like we do for clicking Advanced Filters. Clicking on Advanced Filters bubbles up and then fires the "exitLendFilterExp" method from the lend-filter-exp.mixin.js file.

All of those operations also need to be called with your click. In other words you need to do four things:
Update this: `this.lendFilterExpVersion = 'c';`
Call this:	`this.updateLendFilterExp();`
Track your event...
Then perform your redirect.

These first two steps are necessary to ensure the continuity of the opted out state of the experiment.

Reply:
Thanks Matt, missed some steps there